packages/cache/package.json

```diff
@@ -19,7 +19,7 @@
     "@emotion/sheet": "^1.1.0",
     "@emotion/utils": "^1.0.0",
     "@emotion/weak-memoize": "^0.2.5",
-    "stylis": "^4.0.10"
+    "stylis": "4.0.12"
```

Comment:
I've decided to pin the version of this dependency as it's super crucial for Emotion and changes to it can affect our code in subtle ways. We want to keep this up-to-date with every release of Stylis - but we should always verify that our test suite passes before we can bump this dependency range